### PR TITLE
`ResultExt` with `unwrap_display()` and `expect_display()`

### PR DESCRIPTION
**Pull request by tyranron**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -29,13 +28,14 @@
 )]
 #![warn(clippy::pedantic, missing_docs)]
 // not that hard
-#![allow(clippy::module_name_repetitions)]  // common for extension traits
+#![allow(clippy::module_name_repetitions)] // common for extension traits
```
Comment (tyranron) [inline src/lib.rs:31]: Changes like these are just a result from running `cargo fmt`.

File: src/result.rs [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    /// Basic usage:
+    ///
+    /// ```rust
+    /// # use stdont::ResultExt as _;
+    /// #
+    /// let x: Result<u32, &str> = Ok(2);
+    /// assert_eq!(x.unwrap_display(), 2);
+    /// ```
+    ///
+    /// ```should_panic
```
Comment (GoldsteinE) [inline src/result.rs:35]: I think there’s a way to make assert on panic message?

Comment (GoldsteinE) [reply src/result.rs:35]: Same in the other `should_panic` tests.

Comment (GoldsteinE) [reply src/result.rs:35]: Nope, not possible. Unfortunate.

**Comment by GoldsteinE** [comment]
@tyranron As this was merged before license was added, please confirm that this is available under MIT or Apache 2.0.

**Comment by tyranron** [comment]
@GoldsteinE I do confirm.

**Pull request merged**